docs/app/Components/ComponentDoc/ComponentExamples.js

```diff
@@ -11,7 +11,7 @@ export default class ComponentExamples extends Component {
     const examples = exampleContext.keys()
       .filter(path => path.includes(`/${this.props.name}Examples.js`))
       .map((path, i) => {
-        const Example = exampleContext(path)
+        const Example = exampleContext(path).default
```

Comment:
No more `add-module-exports`, so we need to use the default prop when `require()` is used.  In this case, webpack `require.context()`